src/transformers/models/wav2vec2/modeling_wav2vec2.py

```diff
@@ -68,49 +69,41 @@ def _compute_mask_indices(
     <https://github.com/pytorch/fairseq/blob/e0788f7007a8473a76db573985031f3c94201e79/fairseq/data/data_utils.py#L376>`__.
     """
     bsz, all_sz = shape
```

Comment:
I know the names weren't good previously, but it would be very nice to give better names now. `bsz` -> `batch_size` and `all_sz` -> maybe `mask_length`?